website/docs/how-to/unleash-edge-quickstart.md

```diff
@@ -0,0 +1,536 @@
+---
+title: Unleash Edge Quickstart Guide
+slug: /how-to/unleash-edge-quickstart
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This document helps you get started with [Unleash Edge](https://docs.getunleash.io/reference/unleash-edge) locally.
+
+## Why Unleash Edge
+
+Unleash Edge as a lightweight layer between your SDKs and your Unleash instance.
+
+It exposes the same HTTP interface while providing higher throughput.
+
+<Tabs>
+
+  <TabItem value="performance" label="Performance & UX" default>
+    Unleash Edge helps you reduce latency for flag resolution by running closer to your users. For example, Unleash Edge could run on a global CDN (Content Delivery Network) or as part of your cloud/on-premises infrastructure.
+
+    Setting up one or more Edge nodes helps you distribute traffic and reduce the load on your Unleash instance. By default, Unleash Edge relies on in-memory caching, but you can configure it to use Redis or the local filesystem.
+  </TabItem>
+
+  <TabItem value="security" label="Security & resilience">
+    From a security standpoint, Unleash Edge lets you expose a single, SDK-compatible endpoint without opening your Unleash instance to the public, thus reducing the attack surface.
+
+    At the same time, Unleash Edge provides an additional layer of resilience, so brief upstream hiccups don't disrupt feature delivery.
+
+    Optionally, you can set up multiple layers between your application and the Unleash instance (daisy-chaining). This lets you define the optimal configuration for caching and resilience, while adapting Unleash Edge to your architecture's topology.
+  </TabItem>
+
+  <TabItem value="offline" label="Offline mode">
+    Unleash Edge can also run in [offline mode](https://github.com/Unleash/unleash-edge/tree/main/examples), so it doesn't need a connection to an upstream Unleash instance. This simplifies local development and helps in environments with limited connectivity.
+  </TabItem>
+
+</Tabs>
+
+## Prerequisites
+
+Here's what you need before getting started:
+
+1. An [Unleash instance](https://docs.getunleash.io/quickstart) running locally or remotely (version `4.15` or later)
+2. A valid [API token](https://docs.getunleash.io/reference/api-tokens-and-client-keys) for your Unleash instance
+3. [Docker](https://www.docker.com/get-started/) installed and running
+4. Your preferred Unleash SDK in a [sample app](https://github.com/Unleash/unleash-sdk-examples)
+
+
+## How to run Unleash Edge locally
```

Comment:
For task-based stuff, we typically go for plain form, not question form https://developers.google.com/style/headings

But I'm happy to experiment with it here